src/main/scala/com/codacy/client/bitbucket/v2/service/PullRequestServices.scala

```diff
@@ -50,34 +61,38 @@ class PullRequestServices(client: BitbucketClient) {
       sourceBranch: String,
       destinationBranch: String
   ): RequestResponse[JsObject] = {
-    val url = s"https://bitbucket.org/api/2.0/repositories/$owner/$repository/pullrequests"
+    val pullRequestsUrl = generatePullRequestsUrl(owner, repository)
 
     val payload = Json.obj(
       "title" -> title,
       "source" -> Json.obj("branch" -> Json.obj("name" -> sourceBranch)),
       "destination" -> Json.obj("branch" -> Json.obj("name" -> destinationBranch))
     )
 
-    client.postJson(Request(url, classOf[JsObject]), payload)
+    client.postJson(Request(pullRequestsUrl, classOf[JsObject]), payload)
   }
 
   def postApprove(owner: String, repository: String, prId: Long): RequestResponse[JsObject] = {
-    val url = s"https://bitbucket.org/api/2.0/repositories/$owner/$repository/pullrequests/$prId/approve"
+    val pullRequestsUrl = generatePullRequestsUrl(owner, repository)
+    val url = s"$pullRequestsUrl/$prId/approve"
```

Comment:
Codacy found an issue: [The string literal '${ .. }/${ .. }/approve' appears 2 times in the file.](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)